pyfr/solvers/base/elements.py

```diff
@@ -1,11 +1,26 @@
-from functools import cached_property
+from functools import cached_property, wraps
 
 import numpy as np
 
 from pyfr.nputil import npeval, fuzzysort
 from pyfr.util import memoize
 
 
+def inters_map(meth):
+    @wraps(meth)
+    def newmeth(self, eidx, fidx):
+        nfp = self.nfacefpts[fidx]
+        cmap = (eidx,)*nfp
+
+        mid, rmap, *lda = meth(self, eidx, fidx)
+
+        if lda:
```

Comment:
@FreddieWitherden you might have some idea of a cleaner way to do this. The reason for needing this is the result gets passed to `np.concatenate` so you can't pass back None

Reply:
What methods return four things?  If I recall all of the existing interface methods are expected to return 3-tuples.